src/main/java/org/example/UserService.java

```diff
@@ -0,0 +1,80 @@
+package org.example;
+
+import java.util.*;
```

Comment:
Don't use wildcard imports.
Setup settings for imports (wildcard and import optimization) from wiki: https://www.notion.so/Code-Quality-d260e7578fe045bab7aa8827251d9405?pvs=4